### DISCUSSION TITLE
Movung Glossary in Chapter 0x11-V2-Authentication to App. A

### DISCUSSION THREAD

**Discussion by JoergBruenner** [OWASP/ASVS#1574] [Ideas]
Hi,

the glossary of terms in the middle of the document seems to be a bit unusual. How about moving it to Appendix A?

Regards Jörg Brünner

**Comment by elarlang** [comment]
We have issue (from you) for that discussion:
https://github.com/OWASP/ASVS/issues/1479

In the context of German translation - the change will be done for next release, it means, it does not influence anymore v4.0.3 and this one we need translate like it's written in English.

**Comment by JoergBruenner** [comment]
Your Answer in #1479 sounded to me like "We'll do it in v5.0". I was surprised to find the glossary still in the authentication chapter.

**Comment by elarlang** [comment]
Yes, "We'll do it for v5.0" is still valid.

Logic behind that - there is no reason to update chapter texts before we have finished moving around requirements in the chapter. In other words, (re)writing chapter texts is one of the last things to do.

The same with glossary, there is need to rebuild it, not just update it. Then we can be sure that all required things are covered and not required things are cleaned up.

**Comment by elarlang** [comment]
As the discussion duplicates the issue #1497, I close the discussion and we can continue (if needed) in the issue.
